src/RenderEngineManager.cc

```diff
@@ -93,10 +93,10 @@ class ignition::rendering::RenderEngineManagerPrivate
   public: std::map<std::string, std::string> enginePlugins;
 
   /// \brief Plugin loader for managing render engine plugin libraries.
-  public: ignition::plugin::Loader pluginLoader;
+  public: gz::plugin::Loader pluginLoader;
 
   /// \brief Environment variable which holds paths to look for plugins
-  public: std::string pluginPathEnv = "IGN_RENDERING_PLUGIN_PATH";
+  public: std::string pluginPathEnv = "GZ_RENDERING_PLUGIN_PATH";
```

Comment:
This needs tick-tocking. I think we can follow the example in https://github.com/gazebosim/gz-gui/pull/384. This could come in a separate PR for quicker review.

Reply:
🤘

:https://github.com/gazebosim/gz-rendering/pull/629/commits/ad4901edda747c4a969a2eae72a2e673e166248b

Reply:
Thanks! It was failing to compile for me, so I pushed a fix in [d156275](https://github.com/gazebosim/gz-rendering/pull/629/commits/d15627548bad19d5f9fc01a0b30f6b6e9afd1414) and updated the migration guide.

I checked with the `hello_world_plugin` example that

```
IGN_RENDERING_PLUGIN_PATH=`pwd` ign gazebo shapes.sdf --render-engine HelloWorldPlugin -v 2
```

prints a warning

`[GUI] [Wrn] [RenderEngineManager.cc:477] Found plugin [HelloWorldPlugin] using deprecated environment variable [IGN_RENDERING_PLUGIN_PATH]. Please use [GZ_RENDERING_PLUGIN_PATH] instead.`

And 

```
GZ_RENDERING_PLUGIN_PATH=`pwd` ign gazebo shapes.sdf --render-engine HelloWorldPlugin -v 2
```

doesn't

Reply:
Whoops! Thanks for the change :bow: